.github/workflows/run-cctp-tests.yml

```diff
@@ -18,42 +15,9 @@ env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
-  check-files:
```

Comment:
Tests are run on approval only, we don't need to check for files or hotfix anymore.